### PR TITLE
[doc] update instruction for ait2

### PR DESCRIPTION
**Pull request by sherry-x**
### Description
Update node installation doc

### Test Plan

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1660)

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/using-aws.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Make sure you complete these pre-requisite steps before you proceed:
 1. Set up your AWS account. 
 2. Make sure the following are installed on your local computer:
 
-   * **Aptos CLI**: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md
-   * **Terraform 1.1.7**: https://www.terraform.io/downloads.html
+   * **Aptos CLI 0.2.0**: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/using-aws.md:17]: This doesn't match the CLI version

File: developer-docs-site/docs/nodes/validator-node/using-aws.md [hunk 2]
```diff
@@ -66,11 +66,11 @@ Make sure you complete these pre-requisite steps before you proceed:
 
     module "aptos-node" {
       # Download Terraform module from aptos-labs/aptos-core repo
-      source        = "github.com/aptos-labs/aptos-core.git//terraform/aptos-node/aws?ref=main"
+      source        = "github.com/aptos-labs/aptos-core.git//terraform/aptos-node/aws?ref=testnet"
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/using-aws.md:69]: Probably can remove the double /

**Review by davidiw — APPROVED** [review]

**Review by gregnazario — APPROVED** [review]

File: developer-docs-site/docs/nodes/validator-node/using-aws.md (hunk 1 above)
Comment (sherry-x) [reply developer-docs-site/docs/nodes/validator-node/using-aws.md:17]: Will release and update the link here, the release job is still running

File: developer-docs-site/docs/nodes/validator-node/using-aws.md (hunk 2 above)
Comment (sherry-x) [reply developer-docs-site/docs/nodes/validator-node/using-aws.md:69]: that's actually required for the path to work

**Pull request merged**